latex/epoch.tex

```diff
@@ -308,26 +309,31 @@ \subsection{Rewards Distribution}
 %% Figure - Functions for the Reward Calculation
 %%
 \begin{figure}[htb]
+  \emph{Maximal Reward Function, called $f(s,\sigma)$ in section 6.5.1 of \cite{delegation_design}}
+  %
   \begin{align*}
-      & \fun{maxPool} \in \PParams \to \Coin \to [0, 1] \to [0,1] \to \Coin
-      & \text{max pool reward} \\
-      & \fun{maxPool}~\var{pp}~\var{R}~\sigma~\var{p_r} = \\
-      & ~~~\floor*{
-           \frac{R}{1 + a_0}
-           \cdot
-           \left(\sigma' + p'\cdot a_0\cdot\frac{\sigma' - p'\frac{z_0-\sigma'}{z_0}}{z_0}\right)} \\
+      & \fun{maxPool} \in \PParams \to \Coin \to \unitInterval \to \unitInterval \to \Coin \\
+      & \fun{maxPool}~\var{pp}~\var{R}~\sigma~\var{p_r} =
+          ~~~\floor*{
+             \frac{R}{1 + a_0}
+             \cdot
+             \left(
+               \sigma' + p'\cdot a_0\cdot\frac{\sigma' - p'\frac{z_0-\sigma'}{z_0}}{z_0}
+             \right)} \\
       & ~~~\where \\
       & ~~~~~~~a_0 = \fun{influence}~pp \\
       & ~~~~~~~n_{opt} = \fun{nopt}~pp \\
       & ~~~~~~~z_0 = 1/n_{opt} \\
       & ~~~~~~~\sigma'=\min(\sigma,~z_0) \\
       & ~~~~~~~p'=\min(p_r,~z_0) \\
-      \nextdef
-      & \fun{movingAvg} \in \PParams \to \HashKey \to \mathbb{N} \to \mathbb{R} \\
-      & ~~~\to \Distr \to [0, 1]
-      & \text{moving average} \\
-      & \fun{movingAvg}~\var{pp}~\var{hk}~\var{n}~\var{\overline{N}}~\var{avgs} = \\
-      & \begin{cases}
+  \end{align*}
+
+  \emph{Exponential moving average, see in section 6.5.1 of \cite{delegation_design}}
```

Comment:
maybe constrain typing here, e.g. for \overline(N), also return value should be non-negative, we should add a property for this (could be proven in a mathematical way and quickchecked in the exec spec)